build.gradle

```diff
@@ -9,7 +12,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.2.2'
+        classpath 'com.android.tools.build:gradle:7.2.2'
```

Comment:
`Euphony` used too old version of gradle version. haha
Thanks to update 👍

Reply:
To enable ```compose```, gradle version update was needed :) Thank you!